scripts/verify_windows_ssm_parameters.sh

```diff
@@ -71,6 +74,14 @@ if [[ $ACTION == "verify-ssm-parameter" ]]; then
 
   # Split the comma separated list of regions into an array
   REGIONS=(`echo $REGIONS_TO_PUBLISH | sed 's/,/\n/g'`)
+
+  # AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB is only available if we auth to dockerhub and get it
+  if [[ "$ACCOUNT_ID" == "${classic_regions_account_id}" ]]; then
+    set_dockerhub_version "us-west-2"
+  else
+    set_dockerhub_version "${REGIONS[0]}"
+  fi
```

Comment:
I need to test that this bit actually works in the windows pipeline. I checked that the script to auth to docker hub works.

Reply:
Also we will need to clone the docker hub secret to all sync regions.

Reply:
@rawahars FYI.